### ISSUE TITLE
convert to tflite raise error of "no matching TensorFlow activation found for ReLU"

### ISSUE THREAD

**Issue by sinnuswong** [ultralytics/yolov5#12440]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I got an issue when I try to convert to tflite.
python.exe export.py --data="models/yolov5s-person.yaml"
--weights="runs/train/exp2/weights/best.pt"  --include=saved_model
export: data=models/yolov5s-person.yaml, weights=['runs/train/exp2/weights/best.pt'], imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace
=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False,
topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['saved_model']
YOLOv5  v6.1-174-gc4cb7c6 torch 2.1.0+cpu CPU

Fusing layers...
YOLOv5s-person summary: 213 layers, 7012822 parameters, 0 gradients, 15.8 GFLOPs
****************32

PyTorch: starting from runs\train\exp2\weights\best.pt with output shape (1, 25200, 6) (13.7 MB)
2023-11-29 14:54:25.191730: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library cudart64_110.dll

TensorFlow SavedModel: starting export with tensorflow 2.5.0...

                 from  n    params  module                                  arguments
2023-11-29 14:54:26.595612: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library nvcuda.dll
2023-11-29 14:54:26.621567: E tensorflow/stream_executor/cuda/cuda_driver.cc:328] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device
 is detected
2023-11-29 14:54:26.624278: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: CH3ULTJ660R73
2023-11-29 14:54:26.624406: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: CH3ULTJ660R73
2023-11-29 14:54:26.624680: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network
Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

TensorFlow SavedModel: export failure: no matching TensorFlow activation found for ReLU(inplace=True)

**Comment by sinnuswong** [comment]
it happens here

def activations(act=nn.SiLU):
    # Returns TF activation from input PyTorch activation
    if isinstance(act, nn.LeakyReLU):
        return lambda x: keras.activations.relu(x, alpha=0.1)
    elif isinstance(act, nn.Hardswish):
        return lambda x: x * tf.nn.relu6(x + 3) * 0.166666667
    elif isinstance(act, (nn.SiLU, SiLU)):
        return lambda x: keras.activations.swish(x)
    else:
        raise Exception(f'no matching TensorFlow activation found for PyTorch activation {act}')

seems need a ReLU(inplace=True)
<class 'models.tf.TFConv'>
Conv(
  (conv): Conv2d(3, 32, kernel_size=(6, 6), stride=(2, 2), padding=(2, 2))
  (act): ReLU(inplace=True)
)

how can I change code to fix this?

**Comment by glenn-jocher** [comment]
@sinnuswong consider modifying the code to accommodate the ReLU activation found in the model. You might update the `activations` function to handle the ReLU activation with `inplace=True` or include a case for it in the existing `activations` function. Keep in mind that this workaround could impact the model's behavior. Good luck!

**Comment by sinnuswong** [comment]
@glenn-jocher  thanks , I solved the issue by your advice.

**Comment by glenn-jocher** [comment]
@sinnuswong glad to hear that you were able to solve the issue! If you have any more questions or need further assistance, feel free to ask. Keep up the great work! 😊
